docs/Quick Reference.md

```diff
@@ -23,11 +23,11 @@ This table summarizes the filters and other options available inside a `tasks` b
 | `happens (before, after, on) <date>`<br>`happens (before, after, in) ...`<br>`... YYYY-MM-DD YYYY-MM-DD`<br>`... (last, this, next) (week, month, quarter, year)`<br>`... (YYYY-Www,YYYY-mm, YYYY-Qq, YYYY)`<br>`has happens date`<br>`no  happens date`                                        | `sort by happens`                           | `group by happens`     |                        |
 | **[[Filters#Recurrence\|Recurrence]]**                                                                                                                                                                                                                                                          |                                             |                        |                        |
 | `is recurring`<br>`is not recurring`                                                                                                                                                                                                                                                            | `sort by recurring`                         | `group by recurring`   |                        |
-| `recurrence (includes, does not include) <string>`<br>`recurrence (regex matches, regex does not match) /regex/i`                                                                                                                                                                               |                                             | `group by recurrence`  | `hide recurrence rule` |
-| **[[Filters#Priority\|Priority]]** and **[[Urgency\|urgency]]**                                                                                                                                                                                                                                          |                                             |                        |                        |
-| `priority is (above, below, not)? (low, none, medium, high)`                                                                                                                                                                                                                                    | `sort by priority`                          | `group by priority`    | `hide priority`        |
+| `recurrence (includes, does not include) <string>`<br>`recurrence (regex matches, regex does not match) /regex/i`
```

Comment:
From the github web view, it looks like the text for 'group by recurrence' and 'hide recurrence rule' has disappeared.

Reply:
should be fine now